### DISCUSSION TITLE
Is there a way to connect to host with classic Bluetooth instead of BLE?

### DISCUSSION THREAD

**Discussion by alijani1** [h2zero/NimBLE-Arduino#625] [General]
I am using 8Bitdo sega as host and want to emulate a game controllers on ESP32 to it but it wont pair correctly. The 8Bitdo device does not support BLE connections and only supports devices that use classic bluetooth connectivity.   Is there a way to set the protocol to use classic Bluetooth instead of low energy BLE?

**Comment by h2zero** [comment]
This library does not support classic Bluetooth. You'll need to use the Bluedroid stack.
